### PR TITLE
Introduction of documentation for technical preview.

### PR DESCRIPTION
**Pull request by sgrasley**
This is the first commit for technical preview documentation.

### CODE REVIEW

**Comment by toabctl** [comment]
Why do we have now 2 doc source folders? This is creating so much documentation debt that I would prefer to revert this commit and integrate the written docs properly into what we already had. I know that this will be more work but we could also splitting the commit into smaller parts (eg. the contributor guides can easily be in an extra commit)

**Comment by evrardjp** [comment]
I am confused too.

**Comment by toabctl** [comment]
@sgrasley @jgu17 @matthew-fuller any comments on this?

File: doc/source_tech_preview/contributor.rst [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+* All code requires to go through our :ref:`reviews`.
+* Documentation should be provided with the
+  code directly. See also :ref:`documentation`.
+
+.. _bug_reporting:
+
+Bug reporting process
+---------------------
+
+Bugs should be filed as Github issues.
```
Comment (dirkmueller) [inline doc/source_tech_preview/contributor.rst]: That's not actually true - we want to leverage bugzilla for the public beta program for this.

File: doc/source_tech_preview/deployment/setup-deployer.rst [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+   <username> ALL=(ALL) NOPASSWD: ALL
+
+Add the above line after "#includedir /etc/sudoers.d". replace <username> with
+your username.
+
+
+Configure Ansible
+-----------------
+
+Use ARA (recommended)
```
Comment (dirkmueller) [inline doc/source_tech_preview/deployment/setup-deployer.rst]: why is ARA recommended for non developer builds?

File: doc/source_tech_preview/operations.rst [hunk 3]
```diff
@@ -0,0 +1,293 @@
...
+.. code-block:: console
+
+   kubectl logs -n ${NAMESPACE} ${POD_NAME} -p
+
+.. _caaspoperations:
+
+CaaS Platform Operations
+========================
+
+Disable transactional update for development purposes
```
Comment (dirkmueller) [inline doc/source_tech_preview/operations.rst]: why are we documenting this?

File: doc/source_tech_preview/reference.rst [hunk 4]
```diff
@@ -0,0 +1,333 @@
...
+
+The OpenStack-Helm project provides shell scripts to deploy the helm charts,
+with overrides per context (for example, multinode).
+
+Those shell scripts calling the helm installation include an environment
+variable, to allow users to pass extra arguments.
+
+See for example https://github.com/openstack/openstack-helm/blob/c869b4ef4a0e95272155c5d5dd893c72976753cd/tools/deployment/multinode/100-glance.sh#L49 .
+
+SUSE-ification of the OSH charts when deploying in OSH only mode
```
Comment (dirkmueller) [inline doc/source_tech_preview/reference.rst]: Please don't use the term "SUSE-ification", it is misleading - there is no green paint added.

**Comment by jgu17** [comment]
> Why do we have now 2 doc source folders? This is creating so much documentation debt that I would prefer to revert this commit and integrate the written docs properly into what we already had. I know that this will be more work but we could also splitting the commit into smaller parts (eg. the contributor guides can easily be in an extra commit)

ah, yes, it was a Friday afternoon mistake. We still using the feature branch. The squad already talked about this yesterday that we need to revert it. Now we have comments from Dirk and others, we'll take thosecomments first before revert.

File: doc/source_tech_preview/contributor.rst (hunk 1 above)
Comment (jgu17) [reply doc/source_tech_preview/contributor.rst]: The program meeting decided not to let the tech preview users to file bugs directly, but send issues to the Beta mailing list. Is that aligned to what you expect?

File: doc/source_tech_preview/deployment/setup-deployer.rst (hunk 2 above)
Comment (jgu17) [reply doc/source_tech_preview/deployment/setup-deployer.rst]: copy and pasted from the existing. Agreed it probably should be in the developer guide.

File: doc/source_tech_preview/operations.rst (hunk 3 above)
Comment (jgu17) [reply doc/source_tech_preview/operations.rst]: agreed too.

File: doc/source_tech_preview/reference.rst (hunk 4 above)
Comment (jgu17) [reply doc/source_tech_preview/reference.rst]: "no green paint added." :-)

**Pull request merged**